src/Front/ServiceProvider.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+namespace RocketCDN\Front;
+
+use RocketCDN\Dependencies\League\Container\ServiceProvider\AbstractServiceProvider;
+
+class ServiceProvider extends AbstractServiceProvider {
+	/**
+	 * Services provided by this provider
+	 *
+	 * @var array
+	 */
+	protected $provides = [
+		'cdn',
+		'cdn_subscriber',
+	];
+
+	/**
+	 * Subscribers provided by this provider
+	 *
+	 * @var array
+	 */
+	public $subscribers = [
+		'cdn_subscriber',
+	];
+
+	/**
+	 * Registers the provided classes
+	 *
+	 * @return void
+	 */
+	public function register() {
```

Comment:
If you go with the class extractions above, this would need to be changed to do the injections.